src/components/BlogList/BlogList.css

```diff
@@ -0,0 +1,120 @@
+.blog-container {
+  width: 85%;
+  display: flex;
+}
+.blog-container .blog-list {
+  width: 40%;
+  background-color: #f5f6fa;
+  padding: 30px;
+}
+.blog-container .blog-list .blog-list-header {
+  display: flex;
```

Comment:
Can you wirte a mixin for flex related styles?

Reply:
Flex styles and colors are added. Please look into scss files.